lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -85,24 +85,37 @@
  * @TODO: move into Forge pattern library if approved.
  * Add to the _regions/_container pattern.
  */
-.container__block  {
-  &.-primary {
-    @include media($tablet) {
-      @include span(6 of 12)
-    }
-
-    @include media($desktop) {
-      @include span(8 of 12)
-    }
-  }
+.container {
+  > .wrapper {
```

Comment:
Not needed for merge, but we should fix whatever specificity issue is causing us to need these extra selectors.